HelpSource/Classes/Function.schelp

```diff
@@ -448,6 +448,32 @@ argument:: separately
 For multi channel signals, set whether to use separate value display ranges or not.
 a window to place the plot in. If nil, one will be created for you.
 
+
+method::asBuffer
+Calculates duration in seconds worth of the output of this function, and returns it in a link::Classes/Buffer:: of the number of channels. This method immediately returns a buffer, which takes code::duration:: seconds to become filled with data. Then the action function is called.
+
+argument::duration
+The duration of the function to plot in seconds. The default is 0.01.
+
+argument::server
+The server on which the function is calculated. The default is code::Server.default::.
+
+argument::action
+A function that is called when the buffer is filled. It is passed the buffer as argument.
+
+argument::fadeTime
+A fade in and out time in seconds. (Only) when given, an envelope is applied to the signal to avoid clicks.
```

Comment:
Wondering about the functionality of this... what if we make the default value `0`, and then check if `fadeTime` is == 0? IIRC EnvGen still does a single control period of interpolation if a segment time is 0, so it would be good to catch this case, I think.